addons/website/static/src/snippets/s_avatars/000.scss

```diff
@@ -0,0 +1,53 @@
+.s_avatars {
+    gap: map-get($spacers, 1) clamp(map-get($spacers, 1), calc(var(--avatars-size) * 2), map-get($spacers, 3));
+
+    img.o_avatar, .s_avatars_more {
+        border: inherit;
+        border-radius: inherit;
+
+        // .o_avatar is a component shared across backend and fronted.
+        --Avatar-size: var(--avatars-size, 1rem);
+        width: var(--avatars-size) !important;
+        height: var(--avatars-size) !important;
+
+        &:not(:first-child) {
+            margin-left: calc(-1 * var(--avatars-size) / 4);
+        }
+    }
+
+    .s_avatars_more {
+        // Fallback: can be eventually overridden by the user in edit mode
+        font-size: calc(var(--avatars-size) * 0.3);
+    }
+
+    // Options
+    &.o_avatars_center {
+        &.flex-row {
+            justify-content: center;
+        }
+        &.flex-column {
+            align-items: center;
+        }
+    }
+
+    &.o_avatars_end {
+        text-align: right;
+
+        &.flex-row {
+            flex-direction: row-reverse !important;
+        }
+        &.flex-column {
+            align-items: end;
+        }
+    }
+
+    &.o_first_on_top .o_avatar {
+        z-index: var(--avatar-z-index);
+
+        @for $i from 1 through 4 {
+            &:nth-child(#{$i}) {
+                --avatar-z-index: #{5 - $i};
+            }
+        }
+    }
```

Comment:
Depending on what you do about the z-indexes, don't forget to remove this rule :wink: